### PR TITLE
integrate prettier for *.js code standardization

### PR DESCRIPTION
**Pull request by awitherow**
I saw there was desire in another thread to add prettier, figured I'd go ahead and do so since it is not too difficult 👍 

We need to decide on rules though still.

They can be chosen here: https://github.com/prettier/prettier#options

Merge blocked by PR #40

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +60,14 @@
     "url": "git://github.com/coinbase/gdax-node.git"
   },
   "scripts": {
-    "test": "mocha --ui qunit --bail --reporter list tests/*.js"
+    "test": "mocha --ui qunit --bail --reporter list tests/*.js",
+    "fmt": "node_modules/.bin/prettier --write 'index.js' 'lib/**/*.js' 'tests/**/*.js'",
```
Comment (fb55) [inline package.json]: Let's call it `format`. Also, `node_modules/.bin/` is unnecessary.

Comment (fb55) [inline package.json]: Also, we're currently using four spaces in most places, so let's use that. Also, we want es5 dangling commas.

File: package.json [hunk 2]
```diff
@@ -57,6 +60,14 @@
...
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "node_modules/.bin/prettier --write",
```
Comment (fb55) [inline package.json]: ~~`node_modules/.bin/`~~

File: package.json (hunk 1 above)
Comment (fb55) [inline package.json]: There also isn't a need for quotes around `'index.js'`, and we should be okay just looking for all js files.

**Comment by fb55** [comment]
Thanks a lot for this PR! Not sure whether we shouldn't wait for #40 with this, as it will introduce a lot of merge conflicts.

**Comment by awitherow** [comment]
@fb55 yeah sure thing! I can remove the refactor commit and then we can try to get #40 cleaned up and then we can run it again after merging.

File: package.json (hunk 2 above)
Comment (awitherow) [inline package.json]: I was not 100% that one could use prettier if not installed globally.

File: package.json (hunk 1 above)
Comment (awitherow) [inline package.json]: 👍 thanks for the comments, hadn't officially set up prettier in a project before, only was using the vscode plugin on pre-existing setups.

**Comment by fb55** [comment]
Fixed with #60. Thanks a lot @awitherow for getting this started!

**Comment by awitherow** [comment]
🙇 my pleasure!

**Pull request closed without merging**